docs/UserGuide.md

```diff
@@ -2,1012 +2,1171 @@
+---
+
+## General Features
 
+General features are features that may be used anywhere on CLIpboard, regardless of what page you're on.
 
-### Displaying the home page (Course Page): `home`
+### Showing the home page: `home`
 
-Goes back to the home page, which is the Course Page of CLIpboard.
+Brings you to the home page, which is the `Course Page` of CLIpboard.
+
+**Command Format:**
 
-Format:
 ```
 home
 ```
 
-[Back to list of commands](#commands)
+[Back to list of features](#commands)
+
+### Going back to the previous page: `back`
 
-### Displaying the previous page you were at: `back`
+Brings you back to the previous page you were at.
 
-Goes back to the previous page you were at.
+**Command Format:**
 
-Format:
 ```
 back
 ```
 
-[Back to list of commands](#commands)
+[Back to list of features](#features)
 
 ### Undoing the last command you did: `undo`
+Because everyone makes mistakes.
 
+You may `undo` up to 5 previous commands.
+
+**Command Format:**
 
-Format:
 ```
 undo
 ```
 
-[Back to list of commands](#commands)
+[Back to list of features](#features)
 
-### Opening help window: `help`
+### Help: `help`
+When you're feeling lost or unsure of how to do something, use this command to open the help window.
+There, you can see a list of available commands on your current page, as well as a link to this User Guide.
 
-Opens a window showing the list of commands available in the current page, as well as a link to this User Guide.
+**Command Format:**
 
-Format:
 ```
 help
 ```
 
-[Back to list of commands](#commands)
-
-### Scrolling through command history: `UP` or `DOWN` arrow key
-
-Scrolls through past commands in command box.
-
-- Similar to Linux command-line interface, use the up or down arrow keys to scroll through previously typed commands in the command box.
-- You may use the `LEFT` or `RIGHT` arrow keys to edit the command first.
-- Press `ENTER` to execute the command.
-
-[Back to list of commands](#commands)
+[Back to list of features](#features)
 
 ### Clearing all entries: `clear`
 
-Clears all entries in the list pane of the current page.
+Clears all entries in the list of the current page. Useful for when you've ended the curriculum and you wish to delete all data.
 
-<div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
-After the <code>clear</code> command is executed, all entries in the current page will be deleted.
-</div>
+**Command Format:**
 
-Format:
 ```
 clear
 ```
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-Accidentally cleared your data? Don't worry, try <code>undo</code> command!
+Accidentally cleared your data? Don't worry, just run the <code>undo</code> command!
 </div>
 
+<div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
+Be very careful with this command. Deleted data may not be recovered after closing the application or if you've gone beyond the `undo` limit.
```

Comment:
```suggestion
Be very careful with this command. Deleted data may not be recovered after closing the application or if you've gone beyond the `undo` limit of up to 5.
```

If user don't read this in sequence, they may not know what the `limit` in terms of number of times is, or can misinterpret what `limit` actually means.

Reply:
added